ark/util/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@ark/util",
-	"version": "0.39.0",
+	"version": "0.40.0",
```

Comment:
Revert any changes to package versions as this will trigger release on merge which I want to handle separately

Reply:
Done for `@ark/util` in de532016.

What should I do for the version of `@ark/jsonschema` itself? Is it okay to leave that at `1.0.0`?

Reply:
I would set it to `0.0.1` and also enable `"private"` so I can make sure everything is ready to go before we publish (otherwise it would be published when this PR is merged).

Reply:
Done in 8adda929d